tutorials/publish/index.md

```diff
@@ -213,7 +213,7 @@ You need to specify the following parameters:
 * `SERVICE_DISPLAY_NAME` - Display name of your service. You can choose any name you want. 
 * `PAYMENT_ADDRESS` - Ethereum account which will receive payments for this service. You should set it to your ethereum account. 
 * `SERVICE_ENDPOINT` - Endpoint which will be used to connect to your service.
-* `FIXED_PRICE` - Price in AGI for a single call to your service. We will set the price to 1 COG (remember that 1 AGI = 10^8 COGS).
+* `FIXED_PRICE` - Price in AGI for a single call to your service. We will set the price to 10^-8 AGI (remember that 10^-8 AGI = 1 GOG).
```

Comment:
`GOG -> COG`

We have some other parts of tutorial (7) that uses COGs, maybe we should change them too.

Reply:
We are using COGs to denote smaller amounts of AGI, similar to how satoshi is used. So 1 AGI is not per se one COG. 1   COG = 0.00000001 AGI

Reply:
Ibby, not sure I understand you properly, but I would like to make this change because `snet-cli` gets the value of parameter in AGIs not in COGs.

Reply:
We should use proper scientific notation if we're going to work with AGI. e.g. 1x10^-8 AGI

Personally I find it extremely annoying to work with AGI. I'm tired of counting the number of zeros after a decimal point!

But I agree that the tutorial should match whatever the CLI uses.

Reply:
All CLI commands are set to receive AGI (COG are used only at `call-lowlevel`) but the `service_metadata.json` uses `"price_in_cogs": 100000000`. Maybe this is confusing us.

Reply:
Another way is to specify units after price, for instance 10cogs or 0.00001agi

Reply:
I mean support both COG and AGI in snet-cli interface